sentry/src/main/java/io/sentry/SpanContext.java

```diff
@@ -41,6 +41,8 @@ public class SpanContext implements JsonUnknown, JsonSerializable {
   /** Describes the status of the Transaction. */
   protected @Nullable SpanStatus status;
 
+  protected final @NotNull Map<String, Object> data;
```

Comment:
we're not ser/deser this, is this coming in the follow-up PRs or just missed?

Reply:
nope, i forgot it 😅